dispatcher/worker.go

```diff
@@ -128,7 +129,8 @@ var callConsumer = func(httpClient *http.Client, requestID string, logger zerolo
 		req.Header.Set(headerBrokerPriority, strconv.Itoa(int(job.Priority)))
 		req.Header.Set(headerChannelID, job.Data.Message.BroadcastedTo.ChannelID)
 		req.Header.Set(headerConsumerID, job.Data.Listener.ConsumerID)
-		req.Header.Set(headerConsumerToken, job.Data.Listener.Token)
+		req.Header.Set(worker.consumerConnectionConfig.GetTokenRequestHeaderName(), job.Data.Listener.Token)
+		req.Header.Set(headerUserAgent, worker.consumerConnectionConfig.GetUserAgent())
```

Comment:
Should we add a prefix to the user supplied user-agent value?

Reply:
Not sure we want to do it, I do not see any specific value proposition for this. We could add it to the default user agent though; that would be a separate ticket.